mappings/net/minecraft/util/SystemDetails.mapping

```diff
@@ -0,0 +1,36 @@
+CLASS net/minecraft/class_6396 net/minecraft/util/SystemDetails
+	FIELD field_33852 MEBIBYTE_DIVIDER J
+	FIELD field_33854 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_33855 OPERATING_SYSTEM Ljava/lang/String;
+	FIELD field_33856 JAVA_VERSION Ljava/lang/String;
+	FIELD field_33857 JAVA_VM_VERSION Ljava/lang/String;
+	FIELD field_33858 sections Ljava/util/Map;
+	METHOD method_37120 toProfilerString ()Ljava/lang/String;
```

Comment:
Don't think this is that profiler oriented. Let me check my local stuff real quick

Reply:
At least it is used by profilers. Maybe I'll just call it "toString".

Reply:
Hmm, I called it `collect` locally as this is indeed an action of building a string from the sections.

Btw you shouldn't call it `toString` as that will cause an override of `java.lang.Object#toString()`, which the original method doesn't do.

Reply:
Yeah. (Currently there are two classes that have "toString" mapped to intermediary names. Is that a bug?)